overlord/restart/restart.go

```diff
@@ -56,6 +57,37 @@ const (
 	RestartSystemPoweroffNow
 )
 
+type RestartBoundaryPath int
```

Comment:
maybe s/Path/Direction ? this also needs a doc comment

Reply:
Good point, thank you - renamed and added docs